### PR TITLE
profile table update and formatting

### PR DESCRIPTION
**Pull request by romangraef**

### CODE REVIEW

**Comment by Randium** [comment]
???
How does this change the database's table name to `profiles`?
And why did you just **undo** changes instead of fixing them? Should we not sanitize the user's input just because you want some text one can hover over?

File: management/profile.py [hunk 1]
```diff
@@ -1,44 +1,49 @@
...
+            return '< 13'
+        last_age = 12
+        for next_age in [18, 29, 39, 49, 59, 69, 89, 109, 129]:
+            if self.age <= next_age:
+                return f'{last_age+1}-{next_age}'
+            last_age = next_age
+        return '130+'
+
+    class Meta:
+        db_table = 'profiles'
```
Comment (romangraef) [inline management/profile.py]: @Randium this line changes the db name

**Comment by romangraef** [comment]
nope. not just because of that text. if i want a fucking link in that profile a slash filter wont stop me (i can just put literal SLASHs there which users will have to unescape but it WONT stop anyone). rather it will only annoy users. and why should we sanitize it? if you really think we should block links, then how about you make a big fat poll in #announcements with an @everyone asking "Do you want to be able to have links in your profile :+1: for yes, :-1: for no"

also: if you want links filtered, why don't you filter them in regular chat too? i could spread malicious links in regular chat too..

**Comment by Bentechy66** [comment]
Looks good to me

**Pull request merged**